advance_ai_agents/candidate_analyser/main.py

```
@@ -0,0 +1,197 @@
+                agent = Agent(
+                    description=description_multi,
+                    instructions=instructions_multi,
+                    model=OpenAILike(
+                        id=st.session_state.model_id,
+                        _tool_choice="none",
+                        api_key=st.session_state.Nebius_api_key,
+                        base_url=st.session_state.base_url
+                    ),
+                    name="StrictCandidateEvaluator",
+                    tools=[
+                        ThinkingTools(think=True, instructions="Strict GitHub candidate evaluation"),
```

Pylint (3.3.7)

[convention] Line too long (101/100)

(C0301)